zerver/lib/export.py

```diff
@@ -1598,11 +1659,17 @@ def get_analytics_config() -> Config:
 
     return analytics_config
 
+def get_consented_user_ids(consent_message_id: int) -> Set[int]:
+    return set(Reaction.objects.filter(message__id=consent_message_id, emoji_name="+1").
+               values_list("user_profile", flat=True))
```

Comment:
I made this change to make this more robust against changes in the default emoji name we use:
```
-    return set(Reaction.objects.filter(message__id=consent_message_id, emoji_name="+1").
+    return set(Reaction.objects.filter(message__id=consent_message_id,
+                                       reaction_type="unicode_emoji",
+                                       emoji_code="1fd44").
```